tests/fixture/fr/apps/communications/manifest.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Good point.  For now, I'll fix that as per https://bugzilla.mozilla.org/show_bug.cgi?id=1118946#c32 but I'm open to discussion.